hubstorage/batchuploader.py

```diff
@@ -250,17 +252,21 @@ def __str__(self):
 
 
 def _encode_identity(iter):
-    data = StringIO()
+    data = BytesIO()
     for item in iter:
```

Comment:
Sure. Done.

I handled it in another commit because it was already there before this PR.